configurations/AIK_elections_pipeline.py

```diff
@@ -55,6 +55,7 @@
                     FlowResultConfiguration("AIK_survey_demog", "pool_kenya_location", "location"),
                     FlowResultConfiguration("AIK_survey_demog", "pool_kenya_disabled", "disabled"),
 
+                    FlowResultConfiguration("AIK_pool_invitation_activation", "AIK_pool_invitation_2022", "aik_pool_invitation_2022"),
```

Comment:
Just checking the date on this - is 2022 ok it should be june_2022/summer_2022 (I'm not sure if we will be asking other invitations on this project this year)

Reply:
We will not be asking for other invitations for this project this year. 2022 is ok.